docs/getting_started/discover_ui.md

```diff
@@ -14,15 +14,20 @@ Depending on your location within the Back Office, it can contain the following
 - Global search field that allows you to [find content](../search/search_for_content.md) by checking all searchable Fields
 - Drop-down that changes the site context
 - Notification icon that informs you about incoming assignments, for example, items assigned for [your review](../content_management/workflow_management/editorial_workflow.md#review-queue)
+- App switcher which allows to quickly pivot between applications within the QNTM group ecosystem
```

Comment:
```suggestion
- App switcher which allows to quickly switch between applications within the QNTM group ecosystem
```
To pivot is to turn or rotate, like a hinge. Toggle also woudn't do because you have multiple options.